bitcoin/src/bip32.rs

```diff
@@ -374,6 +567,10 @@ impl<'a> Iterator for DerivationPathIterator<'a> {
 }
 
 impl DerivationPath {
+    /// Try to convert this DerivationPath into a NormalDerivationPath
+    pub fn to_normal(&self) -> Result<NormalDerivationPath, Error> {
+        NormalDerivationPath::try_from(self.0.as_slice())
+    }
```

Comment:
Found more convenient to be more generic over conversion and instantiating into `NormalDerivationPath` making it directly converting from a slice.